cloudca/README.md

```diff
@@ -201,3 +202,35 @@ resource "cloudca_port_forwarding_rule" "web_pfr" {
 - public_ip - the public IP address of this rule
 - private_ip - the private IP address of this rule
 - instance_id - the instance associated with the private IP address of this rule
+
+##cloudca_volume
+Manages volumes. Modifying all fields with the exception of instance_id will result in destruction and recreation
+of the rule.
+
+If the instance_id is updated, where the volume has not yet been attached, the volume will be attached to the instance, where the
+volume is attached to an existing instance, the volume will be detached from the previous instance and attached to the new instance.
+
+###Example usage
+```
+resource "cloudca_volume" "data_volume" {
+	service_code = "compute-east"
+	environment_name = "dev"
+
+	name = "Data Volume"
+	size = "20 GB"
+	storage_tier = "performance"
+	instance_id = "f932c530-5753-44ce-8aae-263672e1ae3f"
+}
+```
+
+###Argument reference
+- service_code - (Required)
+- environment_name - (Required)
+- name - (Required) The name of the volume to be created
+- storage_tier - (Required) Either performance, intermediate or standard
+- size - (Required) The size of the volume. Must be a valid size for the chosen storage tier
```

Comment:
For the record, the solution for this is to accept an integer `size_in_gb`, and find the matching offering within the provider, and send the corresponding disk offering ID back to the server. This should make it easy for us to use custom sizes in the future.